src/applications/pact-act/containers/questions/TernaryRadios.jsx

```diff
@@ -74,36 +64,81 @@ const TernaryRadios = ({
 
   return (
     <>
-      <VaRadio
-        data-testid={testId}
-        onBlur={onBlurInput}
-        className="vads-u-margin-bottom--3"
-        error={(formError && 'Select a response.') || null}
-        hint=""
-        label={h1}
-        label-header-level="1"
-        onVaValueChange={onValueChange}
+      <div
+        className={
+          formError
+            ? 'vads-u-margin-bottom--3 form-question-error'
+            : 'vads-u-margin-bottom--3'
+        }
       >
-        {locationList}
-        <VaRadioOption
-          checked={formValue === responses[0]}
-          label={responses[0]}
-          name={shortName}
-          value={responses[0]}
-        />
-        <VaRadioOption
-          checked={formValue === responses[1]}
-          label={responses[1]}
-          name={shortName}
-          value={responses[1]}
-        />
-        <VaRadioOption
-          checked={formValue === responses[2]}
-          label={responses[2]}
-          name={shortName}
-          value={responses[2]}
-        />
-      </VaRadio>
+        <h1 className="form-question-header" id="form-question">
```

Comment:
Consider: I think it's good practice to use a `paw-` prefix when naming classes and IDs for selectors. This ensures we don't have any chances of conflicts with other code on the page.